tests/braille-specs/sw-ke-g1.yaml

```diff
@@ -0,0 +1,338 @@
+  - ['\x2001', '\x2800']
+  - ['\x2002', '\x2800']
+  - ['\x2003', '\x2800']
+  - ['\x2004', '\x2800']
+  - ['\x2005', '\x2800']
+  - ['\x2006', '\x2800']
+  - ['\x2007', '\x2800']
+  - ['\x2008', '\x2800']
+  - ['\x2009', '\x2800']
+  - ['\x200a', '\x2800']
+  - ['\x2028', '\x2800']
+  - ['\x2029', '\x2800']
+  - ['\x202f', '\x2800']
+  - ['\x205f', '\x2800']
+#####################################################
+# only work forwards
+flags: {testmode: forward}
+tests:
+  - # generic double quotation marks backtranslated as specific open and close
+    - '"Hakika mtu huyu alikuwa Mwana wa Mungu"'
+    - ⠦⠠⠓⠁⠅⠊⠅⠁⠀⠍⠞⠥⠀⠓⠥⠽⠥⠀⠁⠇⠊⠅⠥⠺⠁⠀⠠⠍⠺⠁⠝⠁⠀⠺⠁⠀⠠⠍⠥⠝⠛⠥⠴
+
+  - ['"', '⠦'] # backtranslated as '“' (U+201C LEFT DOUBLE QUOTATION MARK)
+  - [' - ', '⠤⠤'] # backtranslated as '–' (EN DASH)
+  - ['…', '⠤⠤'] # backtranslated as '–' (EN DASH)
+  - ['...', '⠤⠤'] # backtranslated as '–' (EN DASH)
+
+# Typeforms - only supported with testmode 'forward'
+  - # 2 consecutive words italicised
+    - “Hakika mtu huyu alikuwa Mwana wa Mungu”
+    - ⠦⠠⠓⠁⠅⠊⠅⠁⠀⠍⠞⠥⠀⠓⠥⠽⠥⠀⠁⠇⠊⠅⠥⠺⠁⠀⠨⠠⠍⠺⠁⠝⠁⠀⠨⠺⠁⠀⠠⠍⠥⠝⠛⠥⠴
+    - {typeform: {italic: '                         ++++++++       '}}
+
+  - # 3 consecutive words italicised
+    - “Hakika mtu huyu alikuwa Mwana wa Mungu”
+    - ⠦⠠⠓⠁⠅⠊⠅⠁⠀⠍⠞⠥⠀⠓⠥⠽⠥⠀⠁⠇⠊⠅⠥⠺⠁⠀⠨⠨⠠⠍⠺⠁⠝⠁⠀⠺⠁⠀⠠⠍⠥⠝⠛⠥⠴⠨⠄
+    - {typeform: {italic: '                         +++++++++++++++'}}
+
+##########################################
+flags: {testmode: backward}
+tests:
+  - # specific open and close quotation marks
+    - ⠦⠠⠓⠁⠅⠊⠅⠁⠀⠍⠞⠥⠀⠓⠥⠽⠥⠀⠁⠇⠊⠅⠥⠺⠁⠀⠠⠍⠺⠁⠝⠁⠀⠺⠁⠀⠠⠍⠥⠝⠛⠥⠴
+    - “Hakika mtu huyu alikuwa Mwana wa Mungu”
+
+###############################################################
+# work in both directions:
+flags: {testmode: bothDirections}
+tests:
+#######################
+# Letter sign (dots 56)
+# Single letter words - no letter sign wanted
+  - ['u', '⠥']
+  - ['U', '⠠⠥']
+
+# apostrophe single letter - no letter sign wanted
+  - ["Ung'ong'o", '⠠⠥⠝⠛⠄⠕⠝⠛⠄⠕']
+
+# Roman Numerals - letter sign wanted
+  - ['i', '⠰⠊']
+  - ['ii', '⠰⠊⠊']
+  - ['II', '⠰⠠⠠⠊⠊']
+  - ['iii', '⠰⠊⠊⠊']
+  - ['iv', '⠰⠊⠧']
+  - ['v', '⠰⠧']
+  - ['vi', '⠰⠧⠊']
+  - ['vii', '⠰⠧⠊⠊']
+  - ['viii', '⠰⠧⠊⠊⠊']
+  - ['ix', '⠰⠊⠭']
+  - ['x', '⠰⠭']
+  - ['xi', '⠰⠭⠊']
+  - ['xii', '⠰⠭⠊⠊']
+  - ['xiii', '⠰⠭⠊⠊⠊']
+
+# Abbreviations - letter sign only needed in Grade 2 to distinguish from contraction.
+  - ['Yn.', '⠠⠽⠝⠲'] # abbreviation for name Yohana (John) 
+
+####################################
+# Quotation marks
+  -
+    - “Hakika mtu huyu alikuwa Mwana wa Mungu”
+    - ⠦⠠⠓⠁⠅⠊⠅⠁⠀⠍⠞⠥⠀⠓⠥⠽⠥⠀⠁⠇⠊⠅⠥⠺⠁⠀⠠⠍⠺⠁⠝⠁⠀⠺⠁⠀⠠⠍⠥⠝⠛⠥⠴
+  - [' “ ” ', '⠀⠦⠀⠴⠀']
+
+################################# 
+# Dashes/strokes
+  - ['-', '⠤']
+  - ['–', '⠤⠤']
+  - ['—', '⠤⠤⠤']
+  - ['/', '⠸⠌']
+
+#############################
+# Bible verse ranges
+  - ['21:38', '⠼⠃⠁⠼⠉⠓']
+  - ['12:1-8', '⠼⠁⠃⠼⠁⠤⠓']
+  - ['113—118', '⠼⠁⠁⠉⠤⠤⠤⠼⠁⠁⠓']
+  - ['18:29—19:16', '⠼⠁⠓⠼⠃⠊⠤⠤⠤⠼⠁⠊⠼⠁⠋']
+
+# Word contractions in Grade 2 - check not contracted in Grade 1
```

Comment:
If we merge all the tests into a single file, I suggest moving this section to the end, after the corresponding grade 2 tests.

Reply:
Done